docs/griptape-tools/official-tools/email-client.md

```diff
@@ -4,15 +4,68 @@ The [EmailClient](../../reference/griptape/tools/email_client/tool.md) enables L
 
 ```python
 import os
-from griptape.tools import EmailClient
+from griptape.structures import Agent
+from griptape.tools import EmailClient, WebScraper
 
-email_client = EmailClient(
+email_tool = EmailClient(
     smtp_host=os.environ.get("SMTP_HOST"),
     smtp_port=int(os.environ.get("SMTP_PORT", 465)),
     smtp_password=os.environ.get("SMTP_PASSWORD"),
     smtp_user=os.environ.get("FROM_EMAIL"),
     smtp_use_ssl=bool(os.environ.get("SMTP_USE_SSL")),
 )
+
+agent = Agent(
+    tools=[email_tool, WebScraper()],
```

Comment:
Let's move `WebScraper` initialization next to `email_tool` so that all Tools are created in one place.

Reply:
ok!